vendor/modules.txt

```diff
@@ -398,6 +398,7 @@ google.golang.org/grpc/serviceconfig
 google.golang.org/grpc/stats
 google.golang.org/grpc/status
 google.golang.org/grpc/tap
+google.golang.org/grpc/test/bufconn
```

Comment:
Internally we have [HTTPMultibin](https://github.com/grafana/k6/blob/master/lib/testutils/httpmultibin/httpmultibin.go), which should do a similar logic. Can you check if it could be a replacement so we can save from adding a new dependency?

Reply:
IMO it's quite more complex compared to `bufconn`. I'd rather stick with what is officially supported.